core/arch/arm/mm/tee_mm.c

```diff
@@ -332,19 +356,25 @@ tee_mm_entry_t *tee_mm_find(const tee_mm_pool_t *pool, paddr_t addr)
 {
 	tee_mm_entry_t *entry = pool->entry;
 	uint16_t offset = (addr - pool->lo) >> pool->shift;
+	uint32_t exceptions;
 
 	if (addr > pool->hi || addr < pool->lo)
 		return NULL;
 
+	exceptions = cpu_spin_lock_xsave(&((tee_mm_pool_t *)pool)->lock);
```

Comment:
I suppose this a sign that `tee_mm_find()` shouldn't take a `const tee_mm_pool_t *` as argument.